.buildkite/autogenerate_pipeline.py

```diff
@@ -321,10 +321,10 @@ def build(self, input, platform_allowlist):
                 if test_name in tests_to_skip:
                     continue
 
-            # The platform is optional. When it is not specified, we don't add
-            # it to the step so that we can run the test in any environment.
+            # The platform is optional. When it is not specified, it defaults
+            # to `x86_64` to get checked against `platform_allowlist`.
             if not platforms:
-                platforms = [None]
+                platforms = ["x86_64"]
```

Comment:
Can't we make it run on any platform that is available instead (basically what the old comment states)? So that CI execution is potentially finishing faster.

Reply:
Noted, I'll try to work it out :)

Reply:
Updated, PTAL when convenient :)